typescript-deno-plugin/src/index.ts

```diff
@@ -79,6 +79,9 @@ class Plugin implements ts.server.PluginModule {
 
   // determines if a specific filename is Deno enabled or not.
   #fileNameDenoEnabled(fileName: string): boolean {
+    if (process.platform === "win32") {
+      fileName = fileName.replace(/\//g, "\\");
```

Comment:
I think this might break the indexing into `settings?.documents?.[fileName]` below?

Reply:
It fixes it. This is an fsPath.